core/src/main/res/values/arrays.xml

```diff
@@ -1,6 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
+    <string-array name="spnAutoDeleteItems">
+        <item>Global</item>
+        <item>Always</item>
+        <item>Never</item>
```

Comment:
I guess it would. The translators would see another resource and might get a bit confused why these strings are separated from the others.
I'm not even sure if Transifex understands this particular format...